docs/source/user_guide/installation/install-from-source.rst

```diff
@@ -162,22 +162,30 @@ You can use Nox to run the unit tests and example notebooks in isolated virtual
 
 The default command
 
-.. tab:: GNU/Linux, MacOS & Windows
-
-	.. code:: bash
+.. code:: bash
 
-		nox -s tests
+	nox
 
-will run the full test suite (integration and unit tests).
+will run pre-commit, install ``Linux`` dependencies, and run the unit tests.
```

Comment:
```suggestion
will run pre-commit, install ``Linux`` dependencies, and run tests.
```
As suggested in [this comment](https://github.com/pybamm-team/PyBaMM/pull/3084#discussion_r1253653304) we can run all tests by default.

Reply:
I think you misread my comment. I suggested moving the docs around, nothing in particular with running integration tests as the default session.